test/benchgc/README.md

```diff
@@ -38,6 +42,9 @@ python -m benchgc [OPTIONS] --driver [DRIVER] --case [CASE]
 * if driver=pattern, please provide the pre-defined pattern name, such as mlp here
 * if driver is a dialect name, please provide the detail op name to start a single op test
 
+### --entry [str]
```

Comment:
Please provide a default value for entry

Reply:
added